### DISCUSSION TITLE
CI workflow run error

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#873] [Q&A]
I just got an email saying "CI: Some jobs were not successful".  Is this related to the PR I just committed?

https://github.com/raceintospace/raceintospace/actions/runs/9600271362

**Comment by alsed** [comment]
Hi @peyre , the first one is mine. Fix already committed.
The others errors may be related to #869

**Reply by peyre** [reply]
Thanks!

**Reply by kkosciusz** [reply]
Given the CI is operational now, I think we should extend it to the PRs so we can catch issues **before** merging pull requests. I'll see how to establish that.

**Comment by peyre** [comment]
I also got this, just before committing alsed's #874:

https://github.com/raceintospace/raceintospace/actions/runs/9601394149

**Reply by kkosciusz** [reply]
This is most likely a transient issue, you can re-run the workflow.
